### ISSUE TITLE
NW Apollyon allowed second group

### ISSUE THREAD

**Issue by xendral** [LegionDark/Issues#1588]
**Date & Time**: 
10:45pm ish to 11:40pm pst

**FFXI Client Version (use `/ver`)**: 
30170707_0

**Server's Expected Client Version matches, yes/no? (use `$ecv`)**: 
Yes

**Character Name**: 
Bozidar

**Nation**: 
Bastok!

**Job(level)/Sub Job(level)**: 
99thf/49dnc

**NPC or Monster or item Name**: 
n/a

**Zone name**: 
NW Apollyon

**Coordinates (use `$where`)**: 
n/a

**ffxiah.com link (for items issues only)**: 
n/a

**Multi-boxing? (multiple clients on same connection)**: 
no

**Steps To Reproduce / any other info**: 
entered limbus solo, On last floor a party of 2 people were able to join limbus and follow the empty floors up to my final floor. They were not able to engage the monsters even after we joined together. They could see the pool though when I killed things. They also were unable to be hurt by AoE's when they were in range of Meteor.

**Comment by Heepster03** [comment]
So from past experience I believe that as soon as the first mob is engaged, the zone should be inaccessible to anyone else at that point. Sounds like the locking instance part of the code isn't working or implemented.

**Comment by TeoTwawki** [comment]
Yup, what Heepster03 said. DSP has a limbus that is bugged tell heck and back from half implemented features and terribly poor decisions when it was added.

**Comment by TeoTwawki** [comment]
Can try pointing out this *specific* bug to darkstar and seeing if anyone is willing to tackle it separately from   full on rewrite. Don't hold your breath though.
